File: src/com/tyfkda/gawanativeandroid/MyJavaScriptInterface.java
package com.tyfkda.gawanativeandroid;

import android.content.Context;
import android.webkit.JavascriptInterface;
import android.widget.Toast;

public class MyJavaScriptInterface {
  private Context context;
  
  public MyJavaScriptInterface(Context context) {
    this.context = context;
  }

  @JavascriptInterface
  public void showToast(String string) {
    Toast.makeText(context, string, Toast.LENGTH_SHORT).show();
  }
}

File: app/src/main/java/com/tyfkda/gawanativeandroid/MyJavaScriptInterface.kt
package com.tyfkda.gawanativeandroid

import android.content.Context
import android.webkit.JavascriptInterface
import android.widget.Toast

class MyJavaScriptInterface(private val context: Context) {
  @JavascriptInterface
  fun showToast(string: String) {
    Toast.makeText(context, string, Toast.LENGTH_SHORT).show()
  }
}
